src/config.rs

```diff
@@ -77,6 +77,8 @@ pub struct JiraConfig {
     pub fix_versions_field: Option<String>,
     // the field name for where the pending build versions go. expected to be a plain text field
     pub pending_versions_field: Option<String>,
+    // optional name of role to restrict octobot comment visibility. (e.g. "Administrators")
```

Comment:
thought: maybe "Developers" is the better example to give here, since that is likely how we perceive it being used?

Reply:
hmm. i wonder if this means that read-only viewers of jira issues won't see these comments. not sure if that's good or bad.. 🤔 

and are they hierarchical? like "read-only" is lower than "developers" so includes developers?

Reply:
Each project may have its own roles defined. For projects which want to make use of this new feature, the admin would need to configure the specific role and users within it that will be used for visibility restriction.

In my case, I will configure my projects to have a Developers role (including octobot) and only project members in that role will see the comments about PR submissions when browsing Jira.

Reply:
spoke offline, this actually is a global configuration, not per-repo, and we are okay with that for now, and opted for adding a fallback instead. :+1: